packages/foundry/remappings.txt

```diff
@@ -0,0 +1,10 @@
+ds-test/=lib/forge-std/lib/ds-test/src/
```

Comment:
Any reason for adding remapping for other stuff? 

I think we can only keep this right? : 

```
@openzeppelin/contracts/=lib/openzeppelin-contracts/contracts/
```

Reply:
> those are by default, 

Yes, but I think we can remove other stuff right and just keep `@openzeppelin/contracts/=lib/openzeppelin-contracts/contracts/` ? 

Because I just tried keeping only `@openzeppelin/contracts/=lib/openzeppelin-contracts/contracts/` in `remappings.txt` and ran `forge remappings` and I got this : 
![Screenshot 2023-08-03 at 11 00 14 PM](https://github.com/scaffold-eth/scaffold-eth-2/assets/80153681/f1188f97-7aaa-4e20-b45f-fc85c2cfd4b0)

So I think it's picking other default stuff without us needing to mention it, Just trying to keep things minimal 

---

> Someone who is used to foundry will not use @openzeppelin but will use instead lib/openzeppelin

Yeah, but I was just going through [OZ docs](https://github.com/OpenZeppelin/openzeppelin-contracts#foundry-git) and found this : 

![Screenshot 2023-08-03 at 10 55 30 PM](https://github.com/scaffold-eth/scaffold-eth-2/assets/80153681/49a58529-276f-458c-953b-a7a38b0d867d)


I mean in all their tutorials they use `@openzepplin`, also its important that we keep things very similar to `hardhat` and common practices

If `lib/openzeppelin` is too much popular then we add its remapping too, I think we add multiple remapping for same thing 

But its really important that we try to keep things as **minimal** as possible 🙌